capstone-frontend/src/components/Login.jsx

```diff
@@ -22,17 +23,28 @@ export class Login extends Component {
 
   loginResponseSuccess = (response) => {
     this.setState({ isLogin: true });
+    this.setState({ googleUser: response });
     this.setState({ profileObj: response.profileObj });
```

Comment:
This can all be done in a single call to `setState`.

Reply:
Ok I changes it to just be one line.